crashappsec/chalk
# feat(cmd_logout): add `logout` top level command to compliment `login` command

## Issue

Related to issue #50 & PR #55

## Description

This adds a new top level chalk command `chalk logout` that is used to delete any previously saved API tokens, and through doing so causing chalk to be _logged out_ from the API. 
At the code  level this does nothing more than check to see if the `$CHALK_API_KEY` and `$CHALK_API_REFRESH_TOKEN`

## Testing

Testing is straightforward:
* Run: strings ./chalk|grep -i REFRESH and see that there is no API REFRESH key saved into chalk
* Login to API: Run either `chalk setup gen` or `chalk setup login` (if you are running on the branch for PR #55 )
* Run: strings ./chalk|grep -i REFRESH and see that there is now a REFRESH key in chalk
* Run `chalk --trace logout` 
* Run: strings ./chalk|grep -i REFRESH and see that there is no API REFRESH key saved into chalk
